ccd-definition/CaseRoles.json

```diff
@@ -35,6 +35,96 @@
     "JurisdictionID": "PUBLICLAW",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorA]",
+    "Name": "Solicitor A",
+    "Description": "Solicitor A role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorB]",
+    "Name": "Solicitor B",
+    "Description": "Solicitor B role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorC]",
+    "Name": "Solicitor C",
+    "Description": "Solicitor C role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorD]",
+    "Name": "Solicitor D",
+    "Description": "Solicitor D role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorE]",
+    "Name": "Solicitor E",
+    "Description": "Solicitor E role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorF]",
+    "Name": "Solicitor F",
+    "Description": "Solicitor F role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorG]",
+    "Name": "Solicitor G",
+    "Description": "Solicitor G role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorH]",
+    "Name": "Solicitor H",
+    "Description": "Solicitor H role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorI]",
+    "Name": "Solicitor I",
+    "Description": "Solicitor I role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "[solicitorJ]",
+    "Name": "Solicitor J",
+    "Description": "Solicitor J role",
+    "JurisdictionID": "PUBLICLAW",
+    "SecurityClassification": "Public"
```

Comment:
Updates to case roles added in https://github.com/hmcts/fpl-ccd-configuration/pull/2388 (awaiting merged). Needed otherwise case submission would not work.